docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -1,12 +1,9 @@
 # MessageListHeader
 
-The `MessageListHeader` component is a clean, stateless component, that doesn't require a `ViewModel`. It consists of three elements:
+The `MessageListHeader` component is a clean, stateless component, that doesn't require a `ViewModel`. It consists of three internal components by default and it
+allows you to customize the data, design style, action handlers and the internal components.
```

Comment:
> consists of three internal components by default

Why do we include this in the docs if we no longer list what these are? Does this give the reader any information?

Reply:
That's a good point. It used to describe the component and it felt a bit weird without having it here.

Maybe rephrasing this to something like: "It allows you to customize the data, design style, action handlers and the internal components that represent the leading and trailing actions and the title content."